### PR TITLE
MF-20 - Make non-core services optional

### LINKED ISSUE #20: Optional services
Is there a way to do the optional installation of some services? For example, I don't want WS adapter or CoAP so they can be optional during helm install.

### PR DESCRIPTION
**Pull request by blokovi**
Make bootstrap and influxdb reader & writer optional
resolves #20

### CODE REVIEW

File: charts/mainflux/README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The following table lists the configurable parameters and their default values.
 | nats.maxPayload                      | Maximum payload size in bytes that the NATS server will accept             | 268435456    |
 | nats.replicaCount                    | NATS replicas                                                              | 3            |
 | mqtt.broker.persistentVolume.size    | data Persistent Volume size                                                | 5Gi          |
+| influxdb.enabled                     | Should influxdb reader & writer be created                                 | false         |
```
Comment (manuio) [inline charts/mainflux/README.md:34]: You can maybe improve this definition. Also, fix the tabulation of last `|`

File: charts/mainflux/README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ The following table lists the configurable parameters and their default values.
 | nats.maxPayload                      | Maximum payload size in bytes that the NATS server will accept             | 268435456    |
 | nats.replicaCount                    | NATS replicas                                                              | 3            |
 | mqtt.broker.persistentVolume.size    | data Persistent Volume size                                                | 5Gi          |
+| influxdb.enabled                     | Enable influxdb reader & writer                                            | false        |
 
 All Mainflux services can have their `logLevel`, `image.pullPolicy`, `image.repository` and `image.tag` overridden. The names of the services are:
```
Comment (nmarcetic) [inline charts/mainflux/README.md:37]: To get a more user-friendly doc, can we explain that we have Mainflux Core which is a minimalistic set of required Mainflux services and we have optional/add-on services and list them. Explain that by default all add-ons/optional services are disabled and how they can be enabled. Don't save worlds, be user friendly :)

Comment (blokovi) [reply charts/mainflux/README.md:40]: I tried to explain it better and I also added bootstrap as optional service.

Comment (nmarcetic) [reply charts/mainflux/README.md:40]: @blokovi I think it's clear enough.

**Pull request merged**